docs/src/pages/components/data-grid/columns/BasicColumnsGrid.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from 'react';
-import { DataGrid, GridColDef } from '@material-ui/data-grid';
```

Comment:
As far as I remember, the objective to use GridColDef was to help developers find the type. Are you sure it's better without?